template/src/ontology/Makefile.jinja2

```diff
@@ -1556,6 +1556,28 @@ update_docs:
 	mkdocs gh-deploy --config-file ../../mkdocs.yaml
 {%- endif %}
 
+.PHONY: release_changelog
+release_changelog: $(REPORTDIR)/changelog.md $(REPORTDIR)/changelog.csv $(REPORTDIR)/changelog.yaml
+
+$(TMPDIR)/latest_release.obo: $(TMPDIR)
+	wget $(ONTBASE)/$(ONT)-base.owl -O $@.tmp.owl || { \
+		echo "Latest release couldn't be found, creating an empty OBO file instead"; \
+		echo "format-version: 1.2" > $@.tmp.owl; \
+	}
+	$(ROBOT) convert -i $@.tmp.owl --check false -f obo $(OBO_FORMAT_OPTIONS) -o $@ && rm $@.tmp.owl
+
+$(TMPDIR)/current_release.obo: $(ONT)-base.owl
```

Comment:
this is good as it is not at all certain the ontology does indeed have an OBO release.